src/games/strategy/engine/framework/mapDownload/MapListingSource.java

```diff
@@ -0,0 +1,53 @@
+package games.strategy.engine.framework.mapDownload;
```

Comment:
This is an important class that allows us to have 'triplea_maps.xml' be either a local file, or one we download from remote. This class reads a local configuration file that tells us the location of triplea_maps.xml, it then passes it back. DownloadRunner will then check if the source begins with "http", if so it will download the file, otherwise it will read from the specified file path relative to the game root folder